static/styles/left_sidebar.css

```diff
@@ -235,24 +235,14 @@ li.show-more-topics {
         margin-left: 0;
         margin-bottom: 0;
 
-        .active_private_messages_section {
-            background-color: hsl(202, 56%, 91%);
-            border-radius: 4px;
-
-            #more_private_messages_sidebar_title {
-                font-weight: 600;
-            }
-        }
-
         span.fa-group {
             font-size: 90%;
         }
 
         li.pm-list-item {
             position: relative;
-            padding: 1px 22px 1px $far_left_gutter_size;
-            margin: 1px 0;
-    
+            padding: 1px 10px 1px $far_left_gutter_size;
```

Comment:
Aligned `unread_count` pills position in `pm-list-item` through this change in padding.